### PR TITLE
[AAE-463] - change dueDate render component

### PR DESCRIPTION
**Pull request by SilviuCPopa**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [x] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
https://issues.alfresco.com/jira/browse/AAE-463

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: lib/process-services-cloud/src/lib/task/task-header/mocks/task-details-cloud.mock.ts [hunk 1]
```diff
@@ -26,7 +27,7 @@ export const assignedTaskDetailsCloudMock = new TaskDetailsCloudModel(
         'name': 'This is a new task ',
         'description': 'This is the description ',
         'createdDate': 1545048055900,
-        'dueDate': 1545091200000,
+        'dueDate': moment(new Date()).valueOf(),
```
Comment (DenysVuika) [inline lib/process-services-cloud/src/lib/task/task-header/mocks/task-details-cloud.mock.ts]: You are wrapping the date and then unwrapping it back, use `Date.now()` to get the number
```suggestion
        'dueDate': Date.now(),
```

File: e2e/process-services-cloud/task-header-cloud.e2e.ts [hunk 2]
```diff
@@ -30,6 +30,7 @@ describe('Task Header cloud component', () => {
     let basicCreatedDate;
     let completedTask;
     let completedCreatedDate;
+    let dueDate;
```
Comment (DenysVuika) [inline e2e/process-services-cloud/task-header-cloud.e2e.ts]: missing type

**Comment by eromano** [comment]
@SilviuCPopa can you please rebase it?

File: lib/process-services-cloud/src/lib/task/task-header/components/task-header-cloud.component.ts [hunk 3]
```diff
@@ -80,6 +82,7 @@ export class TaskHeaderCloudComponent implements OnInit, OnDestroy, OnChanges {
     ) {
         this.dateFormat = this.appConfig.get('dateValues.defaultDateFormat');
         this.dateLocale = this.appConfig.get('dateValues.defaultDateLocale');
+        this.dateTimeFormat = this.appConfig.get('dateValue.defaultDateTimeFormat');
```
Comment (DenysVuika) [inline lib/process-services-cloud/src/lib/task/task-header/components/task-header-cloud.component.ts]: @eromano should we always rely that app.config has date formats or have some sort of defaults in the code?

**Pull request merged**